src/utils/api.js

```diff
@@ -270,6 +270,13 @@ const upload = (zipPath, sourceZipPath, appDir) => {
 
   const fullZipPath = path.resolve(appDir, zipPath);
   const fullSourceZipPath = path.resolve(appDir, sourceZipPath);
+  const isMissingZip = !fs.existsSync(fullZipPath);
+
+  if (isMissingZip) {
+    return Promise.reject(
+      new Error('Missing a built app. Trying running `zapier push`.')
```

Comment:
Alternatively, we could just `console.error` or `catch` this one level up and `context.line`. I'm in favor of more general error handling in `entry` where this ends up.

Reply:
I think I like that a little more, but it's not a blocker for me

Reply:
You're more in favor of `console.error` right here?